0x/Utilities/TxParameters.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System.Numerics;
+
+namespace ZeroX.Utilities
+{
+    // TODO: Add argument validation
+    public class TxParameters
+    {
+        public TxParameters(BigInteger gasPrice, int gasLimit, BigInteger nonce)
+        {
+            GasPrice = gasPrice;
+            GasLimit = gasLimit;
+            Nonce = nonce;
+        }
+
+        public TxParameters(BigInteger gasPrice, int gasLimit)
+        {
+            GasPrice = gasPrice;
+            GasLimit = gasLimit;
+        }
+
+        public TxParameters(int gasLimit)
+        {
+            GasLimit = gasLimit;
+        }
+
+        public BigInteger GasPrice { get; set; } = - 1;
+        public int GasLimit { get; set; } = -1;
+        public BigInteger Nonce { get; set; } = -1;
```

Comment:
It's maybe better to have these ones all `BigInteger` and default to `null`, because if it's not set take the default values from the Node (that's how Nethereum works).

Reply:
You are right, will fix it